javers-spring-boot-starter-mongo/build.gradle

```diff
@@ -2,9 +2,9 @@ dependencies {
     compile project(':javers-spring')
     compile project(':javers-persistence-mongo')
     //should be on application's classpath when using JaversSpringBootStarterMongo
-    provided 'org.springframework.boot:spring-boot-starter-data-mongodb:1.3.0.RELEASE'
```

Comment:
to jak? teraz w javersie używamy właśnie `optional`(see javers-spring/build.gradle), jeśli mamy przejść na`provided`, to trzeba zmienić w wszystkich modułach.
Mamy dobry powód, żeby to zmieniać?